src/surfaceComponent/Card/Card.tsx

```diff
@@ -0,0 +1,74 @@
+import { PropsWithChildren } from "react";
+import * as React from "react";
+import { Button } from "../../inputs";
+import smallImage from "../../static/images/card/card-small.png";
+import bigImage from "../../static/images/card/card-big.png";
+import * as styles from "./Card.module.css";
+
+interface CardProps {
+  fullWidth?: boolean;
+}
+
+export const Card = ({ fullWidth = false }: PropsWithChildren<CardProps>) => {
+  const handlePrimaryButton = () => {};
+  const handleSecondaryButton = () => {};
+
+  const imageUrl = fullWidth ? bigImage : smallImage;
```

Comment:
**nitpick:** image width should be fluid by default. Card wrapper should limit it's width